src/db/groupCategories.ts

```diff
@@ -0,0 +1,20 @@
+import { pgTable, timestamp, uuid, varchar } from 'drizzle-orm/pg-core';
+import { events } from './events';
+import { relations } from 'drizzle-orm';
+
+export const groupCategories = pgTable('group_category', {
+  id: uuid('id').primaryKey().defaultRandom(),
+  groupCategoryLabel: varchar('group_category_label'),
```

Comment:
lets make this shorter to category_label or just label/title

Reply:
changed it to `groupLabel` and also changed `groupCategoryLabelId` to `groupLabelId`